Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4029,3 +4047,48 @@ def update_integration_context(context, object_keys=None, sync=True):
 
 class DemistoException(Exception):
     pass
+
+
+class ObjectMirror(object):
+    def __init__(self, object, entries):
+        self.object = object
+        self.entries = entries
+
+    def extract_for_local(self):
+        if self.object:
+            demisto.info(f'Updating object {self.object["id"]}')
+            return [self.object] + self.entries
+
+
+class SchemeMappingField:
```

Comment:
I don't need that we need a class for this. Just the function in SchemeTypeMapping will receive `def add_field(self, field_name, field_description):`